website/static/js/pages/login-page.js

```diff
@@ -1,6 +1,25 @@
 /**
  * Login page
  */
-var LogInForm = require('js/signIn');
+var $ = require('jquery');
 
+var LogInForm = require('js/signIn');
 new LogInForm.SignIn('#logInForm');
+
+var SignUp = require('../signUp.js');
+new SignUp('#signUpScope', '/api/v1/register/', $('#campaign').val());
```

Comment:
This is a "nice to have"- it won't break things for the release timeline, but it's not a good habit to get into, and "we copy-pasted it" isn't a defense. It's actually *less* reassuring than if it had only happened once.